pkgs/tummychow/git-absorb/pkg.yaml

```diff
@@ -0,0 +1,2 @@
+packiages:
+  - name: tummychow/git-absorb@0.6.11
```

Comment:
pkg.yaml scaffolded by `cmdx s` should have included not only the latest version but also old versions.

Reply:
c3385c47ab5bdc19bf478273e4fe21cc2beba3f4 Added testdata by `cmdx s tummychow/git-absorb`.